{"sections":[],"identifier":{"url":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/noBidTokenSource","interfaceLanguage":"swift"},"metadata":{"symbolKind":"case","externalID":"s:10CloudXCore0A6XErrorO16noBidTokenSourceyA2CmF","roleHeading":"Case","role":"symbol","title":"CloudXError.noBidTokenSource","modules":[{"name":"CloudXCore"}],"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"noBidTokenSource"}]},"variants":[{"paths":["\/documentation\/cloudxcore\/cloudxerror\/nobidtokensource"],"traits":[{"interfaceLanguage":"swift"}]}],"kind":"symbol","abstract":[{"type":"text","text":"No bid token source."}],"schemaVersion":{"minor":3,"patch":0,"major":0},"hierarchy":{"paths":[["doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore","doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError"]]},"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"noBidTokenSource"}],"platforms":["iOS"],"languages":["swift"]}],"kind":"declarations"}],"references":{"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError/noBidTokenSource":{"type":"topic","abstract":[{"type":"text","text":"No bid token source."}],"kind":"symbol","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"noBidTokenSource","kind":"identifier"}],"identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError\/noBidTokenSource","title":"CloudXError.noBidTokenSource","url":"\/documentation\/cloudxcore\/cloudxerror\/nobidtokensource","role":"symbol"},"doc://com.cloudx.sdk.core/documentation/CloudXCore/CloudXError":{"fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"CloudXError","kind":"identifier"}],"kind":"symbol","navigatorTitle":[{"text":"CloudXError","kind":"identifier"}],"type":"topic","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore\/CloudXError","abstract":[{"text":"An error type that represents the CloudX SDK errors.","type":"text"}],"role":"symbol","url":"\/documentation\/cloudxcore\/cloudxerror","title":"CloudXError"},"doc://com.cloudx.sdk.core/documentation/CloudXCore":{"url":"\/documentation\/cloudxcore","type":"topic","role":"collection","title":"CloudXCore","abstract":[],"kind":"symbol","identifier":"doc:\/\/com.cloudx.sdk.core\/documentation\/CloudXCore"}}}